### PR TITLE
Bump AVS version to 6.4.9

### PR DESCRIPTION
**Pull request by mejdoo**
## What's new in this PR?

Bump AVS version to the latest one 6.4.9

#### APK
[Download build #2830](http://10.10.124.11:8080/job/Pull%20Request%20Builder/2830/artifact/build/artifact/wire-dev-PR3047-2830.apk)

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -9,13 +9,13 @@ apply plugin: 'kotlinx-serialization'
 //region avs
 //TODO: Migrate this configuration to new dependency system
 ext {
-    avsVersion = '6.4.4@aar'
+    avsVersion = '6.4.9@aar'
```
Comment (mejdoo) [inline app/build.gradle:12]: An other PR will follow up to refactor AVS versioning.

**Review by BradleyWire — APPROVED** [review]

**Review by android10 — APPROVED** [review]
:+1:

**Comment by mejdoo** [comment]
build again

**Comment by MacDegger** [comment]
Where IS avs 6.4.9?

Android Studio can't find it ... looking in:

  - https://dl.google.com/dl/android/maven2/com/wire/avs/6.4.9/avs-6.4.9.pom
  - https://dl.google.com/dl/android/maven2/com/wire/avs/6.4.9/avs-6.4.9.aar
  - https://jcenter.bintray.com/com/wire/avs/6.4.9/avs-6.4.9.pom
  - https://jcenter.bintray.com/com/wire/avs/6.4.9/avs-6.4.9.aar
  - https://dl.bintray.com/wire-android/releases/com/wire/avs/6.4.9/avs-6.4.9.pom
  - https://dl.bintray.com/wire-android/releases/com/wire/avs/6.4.9/avs-6.4.9.aar
  - https://dl.bintray.com/wire-android/snapshots/com/wire/avs/6.4.9/avs-6.4.9.pom
  - https://dl.bintray.com/wire-android/snapshots/com/wire/avs/6.4.9/avs-6.4.9.aar
  - https://dl.bintray.com/wire-android/third-party/com/wire/avs/6.4.9/avs-6.4.9.pom
  - https://dl.bintray.com/wire-android/third-party/com/wire/avs/6.4.9/avs-6.4.9.aar
  - https://maven.google.com/com/wire/avs/6.4.9/avs-6.4.9.pom
  - https://maven.google.com/com/wire/avs/6.4.9/avs-6.4.9.aar
  - https://repo.maven.apache.org/maven2/com/wire/avs/6.4.9/avs-6.4.9.pom
  - https://repo.maven.apache.org/maven2/com/wire/avs/6.4.9/avs-6.4.9.aar

**Pull request merged**